qiskit_experiments/library/calibration/fine_drag.py

```diff
@@ -0,0 +1,271 @@
+
+        .. math::
+
+            \Omega_y(t)=-\frac{\lambda_1^2\dot{\Omega}_x(t)}{4\Delta}
+
+        In Qiskit pulse, the definition of the DRAG pulse is
+
+        .. math::
+
+            \Omega(t) = \Omega_x(t) + i\beta\,\dot{\Omega}_x(t)\quad\Longrightarrow\quad
+            \Omega_y(t)= \beta\,\dot{\Omega}_x(t)
+
+        from which we identify the ideal value of :math:`\beta` as :math:`-\lambda^2_1/(4\Delta)`.
+        We now assume that there is a small error :math:`{\rm d}\beta` in :math:`\beta` such
+        that the instantaneous Z-angle error is
+
+        .. math::
+
+            \bar\delta(t) = 2\,{\rm d}\beta\, \Omega^2_x(t)
+
+
+        We can integrate :math:`\bar{\delta}(t)`, i.e. the instantaneous Z-angle rotation error,
+        to obtain the total rotation angle error per pulse, :math:`{\rm d}\theta`:
+
+        .. math::
+
+           {\rm d}\theta = \int\bar\delta(t){\rm d}t = 2{\rm d}\beta \int\Omega^2_x(t){\rm d}t
+
+        If we assume a Gaussian pulse, i.e. :math:`\Omega_x(t)=A\exp[-t^2/(2\sigma^2)]`
+        then the integral of :math:`\Omega_x^2(t)` in the equation above results in
+        :math:`A^2\sigma\sqrt{\pi}`. Furthermore, the integral of :math:`\Omega_x(t)` is
+        :math:`A\sigma\sqrt{\pi/2}=\theta_\text{target}`, where :math:`\theta_\text{target}`
+        is the target rotation angle, i.e. the area under the pulse. This last point allows
+        us to rewrite :math:`A^2\sigma\sqrt{\pi}` as
+        :math:`\theta^2_\text{target}/(2\sigma\sqrt{\pi})`. The total Z angle error per pulse
+        is therefore
+
+        .. math::
+
+           {\rm d}\theta=
+            \int\bar\delta(t){\rm d}t=2\,{\rm d}\beta\,\frac{\theta^2_\text{target}}{2\sigma\sqrt{\pi}}
+
+        Here, :math:`{\rm d}\theta` is the Z angle error per pulse. The qubit population produced by
+        the gate sequence shown above is used to measure :math:`{\rm d}\theta`. Indeed, each
+        gate pair Rp - Rm will produce a small unwanted Z - rotation out of the ZX plane. The total
+        rotation out of the ZX plane is then mapped to a qubit population by the final Post gate.
+        Inverting the relation above yields the error in :math:`\beta` that produced the rotation
+        error :math:`{\rm d}\theta`.
+
+        .. math::
+
+            {\rm d}\beta=\frac{\sqrt{\pi}\,{\rm d}\theta\sigma}{ \theta_\text{target}^2}
+
+    # section: reference
+        .. ref_arxiv:: 1 1612.00858
+        .. ref_arxiv:: 2 1011.1949
+    """
+
+    __analysis_class__ = FineDragAnalysis
+
+    @classmethod
+    def _default_experiment_options(cls) -> Options:
+        r"""Default values for the fine amplitude experiment.
+
+        Experiment Options:
+            repetitions (List[int]): A list of the number of times that Rp - Rm gate sequence
+                is repeated.
+            schedule (ScheduleBlock): The schedule for the plus rotation.
+            normalization (bool): If set to True the DataProcessor will normalized the
```

Comment:
why this is experiment option (this is in analysis option too)?

Reply:
Good catch, should be in analysis. See here: 6a9064e